<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="40" started-at="2018-12-26T20:08:03Z" finished-at="2018-12-26T20:08:03Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="40" started-at="2018-12-26T20:08:03Z" finished-at="2018-12-26T20:08:03Z">
      <class name="TestPackage.NewTest">
        <test-method status="FAIL" signature="f1()[pri:0, instance:TestPackage.NewTest@7921b0a2]" name="f1" duration-ms="13" started-at="2018-12-27T01:38:03Z" finished-at="2018-12-27T01:38:03Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at TestPackage.NewTest.f1(NewTest.java:12)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f1 -->
        <test-method status="PASS" signature="f3(org.testng.ITestResult)[pri:0, instance:TestPackage.NewTest@7921b0a2]" name="f3" is-config="true" duration-ms="1" started-at="2018-12-27T01:38:03Z" finished-at="2018-12-27T01:38:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=f1 status=FAILURE method=NewTest.f1()[pri:0, instance:TestPackage.NewTest@7921b0a2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f3 -->
        <test-method status="PASS" signature="f2()[pri:0, instance:TestPackage.NewTest@7921b0a2]" name="f2" duration-ms="1" started-at="2018-12-27T01:38:03Z" finished-at="2018-12-27T01:38:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f2 -->
        <test-method status="PASS" signature="f3(org.testng.ITestResult)[pri:0, instance:TestPackage.NewTest@7921b0a2]" name="f3" is-config="true" duration-ms="0" started-at="2018-12-27T01:38:03Z" finished-at="2018-12-27T01:38:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=f2 status=SUCCESS method=NewTest.f2()[pri:0, instance:TestPackage.NewTest@7921b0a2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f3 -->
      </class> <!-- TestPackage.NewTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
